index.html

```diff
@@ -0,0 +1,292 @@
+
+      <div class="works-list">
+        <div class="featured-img">
+          <img src="ImgPlaceholder.png" alt="Image placeholder" />
+        </div>
+        <div class="right-block">
+          <h4>Multi-Post Stories</h4>
+          <p class="work-text">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard dummy text ever
+            since the 1500s, when an unknown printer took a standard dummy text.
+          </p>
+          <ul class="tags">
+            <li>css</li>
+            <li>html</li>
+            <li>Bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button type="button" class="button-works">See Project</button>
+        </div>
+      </div>
+
+      <div class="works-list">
+        <div class="right-block">
+          <h4>Multi-Post Stories</h4>
+          <p class="work-text">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard dummy text ever
+            since the 1500s, when an unknown printer took a standard dummy text.
+          </p>
+          <ul class="tags">
+            <li>css</li>
+            <li>html</li>
+            <li>Bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button type="button" class="button-works">See Project</button>
+        </div>
+        <div class="featured-img">
+          <img src="ImgPlaceholder.png" alt="Image placeholder" />
+        </div>
+      </div>
+    </section>
+    <!-- contact-us -->
+    <section id="contact-us">
+      <div class="contact-headline">
+        <h5>Let’s chat!</h5>
+        <p class="contact-text">
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
+      </div>
+
+      <form
+        class="contact-form"
+        action="https://formspree.io/f/xpzbbewp"
+        method="post"
+      >
+        <div class="input-area">
+          <div class="user-name">
+            <label for="name"></label>
+            <input
+              type="text"
+              id="name"
+              name="user_name"
+              placeholder="Full name"
+              maxlength="30"
+            />
+          </div>
+
+          <div class="mail-area">
+            <label for="mail"></label>
+            <input
+              type="email"
+              id="mail"
+              name="user_email"
+              placeholder="Email Address"
+            />
+          </div>
+
+          <div class="message-area">
+            <textarea
+              id="message"
+              name="user_message"
+              placeholder="Write your message here"
+              maxlength="500"
+            ></textarea>
+          </div>
+        </div>
+
+        <button class="button-submit" type="submit">Send</button>
```

Comment:
- [ ] Kindly remove the black border for this buttons and make the text color white as Figma design.
### your design
![his](https://user-images.githubusercontent.com/81727988/171836571-a4eabd34-ef71-470b-a9e9-0778dd9bb3ce.png)
### Figma design
![figma](https://user-images.githubusercontent.com/81727988/171836596-caf22f5b-d1c4-42c5-9116-cf9261b856e0.png)